web/src/features/charts/tooltipCalculations.ts

```diff
@@ -90,10 +90,9 @@ export function getExchangeTooltipData(
 ) {
   const { zoneKey, exchangeCo2Intensities, exchangeCapacities } = zoneDetail;
 
-  const co2Intensity = exchangeCo2Intensities[exchangeKey];
-
-  const exchangeCapacityRange = (exchangeCapacities || {})[exchangeKey];
-  const exchange = (zoneDetail.exchange || {})[exchangeKey];
+  const co2Intensity = exchangeCo2Intensities?.[exchangeKey];
+  const exchangeCapacityRange = exchangeCapacities?.[exchangeKey];
+  const exchange = zoneDetail?.exchange?.[exchangeKey];
```

Comment:
It's possible the first optional chain is not needed on this row but I thought it safer to include it than leave it out.